test/ApiFactoryTest.php

```diff
@@ -160,8 +163,10 @@ public function testCreateRestService()
         $this->assertEquals($docConfig['Test\V1\Rest\FooBar\Controller']['description'], $service->getDescription());
 
         $fields = $service->getFields();
```

Comment:
On current master, `$service->getFields()` requires an argument. As such, right now ,this test fails, and I've not figured out what value to provide here. [Master uses the argument "input_filters"](https://github.com/zfcampus/zf-apigility-documentation/blob/599c1effb654084b74df3373d056b6923b895a9e/test/ApiFactoryTest.php#L162). I've tried "input_filter", "foogoober", and "foofoogoober", but none appear to be valid indices.